openquakeplatform/openquakeplatform/test/isc_test.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+import time
+import sys
+import unittest
+
+# from IPython.core import ultratb
+# sys.excepthook = ultratb.FormattedTB(mode='Verbose',
+#      color_scheme='Linux', call_pdb=1)
+
+from openquakeplatform.test import pla
+from openquakeplatform.test.utils import *
+
+
+class IscTest(unittest.TestCase):
+    def isc_test(self):
+        pla.navigate('explore')
+
+        #<li>
+        #<a href="/maps/23">
+        enter_button = pla.xpath_finduniq(
+            "//li/a[@href='/maps/23' and normalize-space(text()) = 'Global "
```

Comment:
Just to be sure, I guess the map is number 23 because we are using some fixtures to populate the db. Are we sure that the number will never change if we change the fixtures?